zyte_common_items/pipelines.py

```diff
@@ -53,3 +54,87 @@ def __init__(self):
 
     def process_item(self, item, spider):
         return ae.downgrade(item)
+
+
+class DropLowProbabilityItemPipeline:
+    """This pipeline drops an item if its probability, defined in the settings,
+    is less than the specified threshold.
+
+    There are two ways that the thresholds can be defined per item type using
+    the ``ITEM_PROBABILITY_THRESHOLDS`` setting
+    - with paths to Item classes,
+    - with Item classes.
+    Additionally, for Item's classes without their own threshold, you can set a default value.
+
+    The example of using ITEM_PROBABILITY_THRESHOLDS:
+
+    .. code-block:: python
+
+        from zyte_common_items import Article
+
+        ...
+
+
+        ITEM_PROBABILITY_THRESHOLDS = {
+            Article: 0.2,
+            "zyte_common_items.items.Product": 0.3,
+            "default": 0.15,  # if not provided, DEFAULT_ITEM_PROBABILITY_THRESHOLD is used
+        }
+        ...
+    """
+
+    DEFAULT_ITEM_PROBABILITY_THRESHOLD = 0.1
```

Comment:
💄 Just DEFAULT_THRESHOLD? The pipeline is already named DropLowProbabilityItemPipeline, so shorter name could be ok.